module/components/AppBar/AppBar.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Navbar as ReactstrapNavbar } from 'reactstrap';
+
+const propTypes = {
+    /**
+     * Useful to extend the style applied to components.
+     */
+    classes: PropTypes.string,
```

Comment:
we need to get className instead classes

Reply:
Done